### PR TITLE
Minor spelling error

### PR DESCRIPTION
**Pull request by jraval**
The title had a minor spelling error that I fixed.

# Description

This was a PR that had already been approved I saw a spelling error in the title of the ReadMe file for the document and I decided to change it.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change may require a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [ ] No
- [x] Not necessary

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts
- [x] N/A

### CODE REVIEW

**Comment by jacob-torres** [comment]
Nice catch! It's good to fix simple and noticeable mistakes like this for the credibility of the project. I'll be on the lookout for similar mistakes in documentation and fix them going forward.

File: structured_experiments/guidelines.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Guidlines for Datascience and Machine learning experimentation
+# Guidelines for Datascience and Machine Learning Experimentation
```
Comment (bofori20) [inline structured_experiments/guidelines.md:1]: Good catch.

**Review by bofori20 — APPROVED** [review]
We can also do with a great pair of eyes. Good catch!

**Review by ragerdude — APPROVED** [review]
Nice PR. Very atomic.

**Pull request merged**